### ISSUE TITLE
Does hunspell/spylls have facility to change the root of the word?

### ISSUE THREAD

**Issue by exander77** [hunspell/hunspell#763]
Does hunspell/spylls have facility to change the root of the word when making word forms?

I am looking through cs_CZ and a lot of words are directly written there in all forms.

Like `stůl` (table):
```
stole
stolech
stolem
stoletím
stolu
stolů
stolům
stoly
stůl
```
Which seems highly inefficient to me.
But it may be that there is no facility to implement character transform as `ů` is becoming `o`.
Or is there something in hunspell format that can handle this?

Basically, every male gender word with `ů` gets transformed like this into `o` form. And there are several transformations like these in Czech language.

**Comment by ssvb** [comment]
You can have affixes that strip `"ůl"` and then append `"oly"`, `"olům"`, etc. This way you get one entry in the `.dic` file for all the wordforms you listed in your comment. And it may be also possible to use two-fold suffixes stripping to reduce excessive repetition/size of the affix file. However two-fold suffixes stripping makes spellchecking slower in general and may be not a good choice for Slavic languages.
